lib/mintacoin/accounts/accounts.ex

```diff
@@ -54,12 +59,27 @@ defmodule Mintacoin.Accounts do
     end
   end
 
-  @spec create_db_record :: {:ok, Account.t()} | {:error, error()}
-  def create_db_record do
+  @spec create_db_record(customer :: customer()) :: {:ok, account()} | {:error, error()}
+  def create_db_record(customer_id) do
     signature_fields = Keypair.build_signature_fields()
 
+    %{
+      address: address,
+      encrypted_signature: encrypted_signature,
+      seed_words: seed_words,
+      signature: signature
+    } = signature_fields
+
+    changeset = %{
+      customer_id: customer_id,
+      address: address,
+      encrypted_signature: encrypted_signature,
+      seed_words: seed_words,
+      signature: signature
+    }
```

Comment:
This can be improved like so:

```suggestion
    changeset = 
        Keypair.build_signature_fields()
        |> Map.merge(%{customer_id: customer_id})
```